rpc/src/v1/types/optionals.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright 2015, 2016 Ethcore (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+use serde::{Serialize, Serializer};
+use serde_json::Value;
+
+#[derive(Debug)]
+pub enum OptionalValue<T> where T: Serialize {
```

Comment:
`Option::None` is not serialized at all, whereas we need to serialize value to `null`.

eg

```rust
struct Astruct {
  a: Option<usize>
}

let x = Astruct { a: None };
assert_eq!(serde_json.to_string(&x).unwrap(), "");

struct Bstruct {
  a: OptionalValue<usize>
}

let x = Bstruct { a: Null };
assert_eq!(serde_json.to_string(&x).unwrap(), "null");

```